tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServicePossibleValuesService.java

```diff
@@ -28,15 +30,29 @@
 
 public class ServicePossibleValuesService extends PossibleValuesService
 {
+
     @Override
     public Status problem( final Value<?> value )
     {
         return Status.createOkStatus();
     }
-   
+
     @Override
     protected void compute( final Set<String> values )
     {
-        values.addAll( Arrays.asList( NewLiferayModuleProjectOpMethods.getServices() ) );
+        try
```

Comment:
Hi greg, Because the getService method directly call Lovetti's method getting OSGI service from Running Server or static service list. I discussed with Lovetti that he was doing this section work that he would add the connection checkong in his method internal. If you think this isn't enough, do you mean we need to check the server running status in IDE?